plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestHivePlans.java

```diff
@@ -165,7 +165,7 @@ public void testPrunePartitionLikeFilter()
                                                 project(
                                                         filter("\"like\"(L_STR_PART, \"$like_pattern\"('t%'))",
                                                                 tableScan("table_str_partitioned", Map.of("L_INT_COL", "int_col", "L_STR_PART", "str_part"))))),
-                                        exchange(LOCAL, REPARTITION,
```

Comment:
you could also use `exchange(io.trino.sql.planner.assertions.PlanMatchPattern...)`. It's not really the point of this test to match exchange type.